torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -441,6 +441,34 @@ RegisterOperators reg(
            stack->emplace_back(result);
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "prim::AutogradAllZero(...) -> bool",
+         [](Stack* stack) {
+           auto num_inputs = pop(stack).toInt();
+           bool result = false;
```

Comment:
This is kind of a nit/stray thought but, at least per python `any` of an empty sequence returns False and `All` of an empty sequence returns true.
```
>>> any(())
False
>>> all(())
True
```